Makefile

```
@@ -0,0 +1,14 @@
+all: build-test check
+
+build-test: clean
+	@docker build -t diff-shellcheck-test-image test/
+
+check:
+	@docker run -dt -v $(pwd):/home --name diff-shellcheck-test diff-shellcheck-test-image
```

```suggestion
	@docker run -dt -v "$PWD:/home" --name diff-shellcheck-test diff-shellcheck-test-image
```

`$PWD` is already provided by shell and let's quote it so that it also works for paths that contain whitespaces, asterisks, etc.

---

Thank you, updated.